Jenkinsfile

```diff
@@ -174,5 +174,45 @@ pipeline {
         }
       }
     }
+
+    stage('Run Enterprise Pipeline') {
+      when {
+        expression {
+          release_branches.contains(BRANCH_NAME) && params.CREATE_RELEASE == false
+        }
+      }
+      steps {
+        build job: "frontend/dcos-ui-ee-pipeline/" + env.BRANCH_NAME.replaceAll("/", "%2F"), wait: false, propagate: false
+      }
+    }
+  }
+
+  post {
```

Comment:
I think this should really have been a seperate ticket. I am okay with merging this right now, but please open a separate ticket for these bigger changes. This has nothing to do with "wrong paths in some scripts", nor does triggering the enterprise pipeline. I don't want to stay in the way of progress and I know how much work went into this solution, so please let's not get into this situation the next time.